src/applications/simple-forms/20-10207/components/FileField.jsx

```diff
@@ -509,137 +515,141 @@ const FileField = props => {
-                        onBlur={onBlur}
-                        registry={indexedRegistry}
-                        disabled={props.disabled}
-                        readonly={props.readonly}
-                      />
-                    </Tag>
-                  )}
-                {!hasErrors &&
-                  !showPasswordInput &&
-                  uiOptions.attachmentName && (
-                    <Tag className="schemaform-file-attachment review">
-                      <SchemaField
-                        name="attachmentName"
-                        required
-                        schema={itemSchema.properties.name}
-                        uiSchema={getUiSchema(uiOptions.attachmentName)}
-                        errorSchema={attachmentNameErrors}
-                        idSchema={attachmentNameSchema}
-                        formData={formData[index].name}
-                        onChange={value => onAttachmentNameChange(index, value)}
-                        onBlur={onBlur}
-                        registry={indexedRegistry}
-                        disabled={props.disabled}
-                        readonly={props.readonly}
-                      />
-                    </Tag>
-                  )}
-                {!file.uploading &&
-                  hasErrors && (
-                    <span className="usa-input-error-message" role="alert">
-                      <span className="sr-only">Error</span> {errors[0]}
-                    </span>
-                  )}
-                {showPasswordInput && (
-                  <ShowPdfPassword
-                    file={file.file}
-                    index={index}
-                    onSubmitPassword={onSubmitPassword}
-                    passwordLabel={content.passwordLabel(file.name)}
-                    uswds={uswds}
-                  />
-                )}
-                {!formContext.reviewMode &&
-                  !isUploading && (
-                    <div className="vads-u-margin-top--2">
-                      {hasErrors &&
-                        enableShortWorkflow && (
-                          <va-button
-                            name={`retry_upload_${index}`}
-                            class="retry-upload vads-u-width--auto vads-u-margin-right--2"
-                            onClick={getRetryFunction(
-                              allowRetry,
-                              index,
-                              file.file,
-                            )}
-                            label={
-                              allowRetry
-                                ? content.tryAgainLabel(file.name)
-                                : content.newFile
-                            }
-                            text={retryButtonText}
-                            uswds={uswds}
-                          />
-                        )}
+              <li key={index} id={fileListId}>
+                <VaCard className={itemClasses}>
```

Comment:
Only new change here is wrapping this whole block in VaCard. contents should be identical.